c2corg_ui/format/ltag.py

```diff
@@ -16,18 +16,27 @@
 class LTagPreprocessor(Preprocessor):
     """ Preprocess lines to clean LTag blocks """
 
+    """ Supported tags are L#, R# """
+    RE_LTAG_UNSUPPORTED = re.compile(r'[L|R]#[^\s|]+')
```

Comment:
I am not sure to understand your regexp then: doesn't it catch strings such as ``L# |`` or ``R# |``?